python/ray/autoscaler/_private/gcp/config.py

```diff
@@ -106,7 +118,7 @@ def _validate_tpu_config(node: dict):
         generation_pattern = re.compile(r"^V\d+[a-zA-Z]*$")
         topology_pattern = re.compile(r"^\d+x\d+(x\d+)?$")
 
-        if not generation_pattern.match(generation):
+        if generation != "V5LITE_POD" and not generation_pattern.match(generation):
```

Comment:
Is this case sensitive?

Reply:
The original string from the proto should always be all caps.